### PR TITLE
Fix Bar Chart Panel & Update Grafana Version

### PR DESCRIPTION
**Pull request by karansohi**
##### Checklist

- [x] Tested in playground or other setup
- [ ] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [ ] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

### Summary by CodeRabbit

- New Feature: Added a new parameter `unit` to the request queue duration bar chart, allowing users to see duration values in milliseconds. This enhances the chart's clarity and precision.
- Improvement: Updated the bar chart panel and time series panel to ensure the correct data source ('prometheus') is used and to provide more flexibility in specifying query options. This change improves the reliability of data queries and allows for more customized data visualization.

### CODE REVIEW

**Comment by karansohi** [comment]
Screenshot 2023-10-17 at 10 26 14 AM
Screenshot 2023-10-17 at 10 26 25 AM
Screenshot 2023-10-17 at 10 26 42 AM

File: blueprints/grafana/panels/request_queue_duration_bar.libsonnet [hunk 1]
```diff
@@ -9,7 +9,8 @@ function(cfg) {
                                          'topk(10, (sum by(workload_index) (increase(request_in_queue_duration_ms_sum{%(filters)s}[$__range])) ) / ((sum by(workload_index) (increase(request_in_queue_duration_ms_count{%(filters)s}[$__range])) )) != 0)',
                                          stringFilters,
                                          instantQuery=true,
-                                         range=false),
+                                         range=false,
+                                         unit=null),
```
Comment (tanveergill) [inline blueprints/grafana/panels/request_queue_duration_bar.libsonnet:13]: Shouldn't the unit be time (ms)?

**Comment by karansohi** [comment]
Screenshot 2023-10-17 at 1 36 00 PM

File: blueprints/grafana/panels/request_queue_duration_bar.libsonnet (hunk 1 above)
Comment (karansohi) [reply blueprints/grafana/panels/request_queue_duration_bar.libsonnet]: @tanveergill Done with the change

**Review by tanveergill — APPROVED** [review]

**Pull request merged**